website_sale_qty/wizards/website_config_settings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html) -->
+
+<odoo>
+    <record id="view_website_config_settings" model="ir.ui.view">
+        <field name="name">Website Settings - Implicit Price Tier</field>
+        <field name="model">website.config.settings</field>
+        <field name="inherit_id" ref="website.view_website_config_settings"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='delivery']" position="inside">
```

Comment:
I just pushed up a fix for this. The original code was inheriting from the parent view in ``website``, which should work as long as the ``website_sale`` changes are applied first, and this seems like the expected order since ``website_sale`` is a depend for this module. I wonder why this wasn't happening in your scenario. In any event, the new approach should be less brittle.